unittests/util/TestHttpServer.cpp

```diff
@@ -19,19 +19,16 @@
 
 #include "util/TestHttpServer.hpp"
 
-#include <boost/asio/buffer.hpp>
 #include <boost/asio/detached.hpp>
 #include <boost/asio/io_context.hpp>
 #include <boost/asio/ip/address.hpp>
 #include <boost/asio/ip/tcp.hpp>
 #include <boost/asio/socket_base.hpp>
 #include <boost/asio/spawn.hpp>
-#include <boost/beast.hpp>
```

Comment:
This actually broke `async_read` compilation on my local env. Interesting 🔢

Reply:
cc @kuznetsss

Reply:
Yeah, it is not perfect, but overall it is quite helpful.